assets/map.js

```diff
@@ -81,63 +81,68 @@ async function loadStationMarkersFromArray(stationsArray) {
     }
     
     let featuresArray = [];
-    let featureID = 0;
-
-    for (var station of stationsArray) {
-
-        // Don't show deactivated stations
-        if (station.docks === 0)
-            continue;
 
+    for (const [featureID, station] of stationsArray.entries()) {
```

Comment:
O featureID não é uma propriedade do objeto station. 
Ele também vai subindo desta forma?

Reply:
Quando fazes Array.entries ele devolve um tuple em que o primeiro elemento é o índice e o segundo é o elemento em si, portanto podes ter um for tradicional com o índice e um for-each na mesma linha

Reply:
brutal! mais uma para a caixinha do conhecimento sdfjkhfhdsdfkjhdsfk